### PR TITLE
MOTECH-1710: ScheduleTracking javadoc repository package

### PR DESCRIPTION
**Pull request by tstalka**

### CODE REVIEW

File: schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/AllEnrollments.java [hunk 1]
```diff
@@ -10,36 +10,82 @@
...
+ * Implementation of DAO interface that utilizes a MDS back-end for retrieving instances.
+ */
 @Repository
 public class AllEnrollments {
 
     @Autowired
     private EnrollmentDataService enrollmentDataService;
 
+    /**
+     * Returns all enrollments stored int he database.
```
Comment (pgesek) [inline schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/AllEnrollments.java]: in the

File: schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/AllEnrollments.java [hunk 2]
```diff
@@ -10,36 +10,82 @@
 import java.util.HashSet;
 import java.util.List;
 
+/**
+ * Implementation of DAO interface that utilizes a MDS back-end for retrieving instances.
```
Comment (pgesek) [inline schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/AllEnrollments.java]: This does not implement any interfaces. Just say that this is a DAO that utilizes..

File: schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/TrackedSchedulesJsonReader.java [hunk 3]
```diff
@@ -4,7 +4,26 @@
...
+/**
+ * <code>TrackedSchedulesJsonReader</code> interface provide methods for creating
+ * {@link org.motechproject.scheduletracking.domain.json.ScheduleRecord} form json data and json files.
+ */
 public interface TrackedSchedulesJsonReader {
+
+    /**
+     * Builds schedule record from the given json data.
+     *
+     * @param schduleJson the json data with schedule definition
```
Comment (pgesek) [inline schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/TrackedSchedulesJsonReader.java]: the schedule definition

File: schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/TrackedSchedulesJsonReader.java [hunk 4]
```diff
@@ -4,7 +4,26 @@
...
 import java.util.List;
 
+/**
+ * <code>TrackedSchedulesJsonReader</code> interface provide methods for creating
+ * {@link org.motechproject.scheduletracking.domain.json.ScheduleRecord} form json data and json files.
+ */
 public interface TrackedSchedulesJsonReader {
+
+    /**
+     * Builds schedule record from the given json data.
```
Comment (pgesek) [inline schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/TrackedSchedulesJsonReader.java]: a schedule record

**Comment by pgesek** [comment]
see inline comments

**Comment by tstalka** [comment]
test this please

**Pull request merged**